README.md

```diff
@@ -40,3 +40,11 @@ Bokeh App
 2. Install the python requirements: `pip install -r requirements.txt`
 3. Download data modules: `git submodule update --init --recursive`
 4. Run bokeh server from top dir of the repo: `bokeh serve --show water`
+
+### Docker version
+
+1. Pull from Docker Hub: `docker pull turinginst/chance-water`
+2. Run the container `docker run -p 5006:5006 -it turinginst/chance-water`
```

Comment:
I don't think we should run this with either the '-i' or '-t' flags. The app isn't interactive. Perhaps '-d' would be more appropriate to let the app run in the background until you stop it.

Reply:
agreed